### ISSUE TITLE
Send lightgbm parameters?

### ISSUE THREAD

**Issue by jmrichardson** [mljar/mljar-supervised#461]
Hi, is it possible to send parameters to a specific model?  I am currently just using lightgbm and would like to set the following:
`
is_unbalance=True`

And send to fit:
`
sample_weight = weights`

Thank you

**Comment by pplonski** [comment]
@jmrichardson there is no such functionality, but I will add it.

**Comment by jmrichardson** [comment]
Great news!  It would also be great to be able to set default parameters per model.  I have a pretty unique dataset and the default parameters don't quite work well for initial start.  Thank you

**Comment by pplonski** [comment]
@jmrichardson good idea. Looks that there should be option to set 3 types of parameters:
- additional parameters for algorithms (not tuned, just passed to the algorithm)
- default parameters for algorithm (that will be used in first model training),
- new range of parameters that will be optimized for algorithm

**Comment by jmrichardson** [comment]
Hi @pplonski ,  wanted to see if this is on the roadmap and when you think it might be available?  Thanks for your consideration.

**Comment by alitirmizi23** [comment]
Any idea if we can do this now? with automl and having custom parameters per model?

**Comment by pplonski** [comment]
Hi @alitirmizi23! Currently, all my efforts are focused on the https://github.com/mljar/mercury framework for converting notebooks to web apps. I hope I can built sustainable bussiness around Mercury and use founds to develop MLJAR AutoML.

I do provide commercial support for MLJAR AutoML. I can implement this feature if needed by a commercial client. Licenses start at $1000/year.

The alternative is to implement this yourself or wait - from time to time I have interns that can code this.

**Comment by alitirmizi23** [comment]
Hey @pplonski, I understand. 
Maybe I'll try to implement myself.
as far as license goes, we already have another enterprise automl framework. To your credit, i have found MLJAR Supervised to be one of the best open source autoML frameworks available

**Comment by pplonski** [comment]
@adrianblazeusz could you please take a look at this issue?
